### ISSUE TITLE
Problem with defaultFn 'now' in Model

### ISSUE THREAD

**Issue by FelipeAguayo** [strongloop/loopback#3114]
### Bug/Feature request

I installed loopback v3.2.1 with strongloop (A LoopBack API server with local User auth) and performed some tests. When creating elements with 'create' and using 'defaultFn:' now "'as the default value in the model's property:
  ```javascript 
Role.create([
    {name: 'ADMIN'},
    {name: 'APP'},
    {name: 'USER'},
  ], function(err, role) {
    if (err) throw err;
    console.log('Roles created!');
  });
```

I found these results in the response json (see created and modified property).

![image](https://cloud.githubusercontent.com/assets/9990531/22034033/9a2ea3dc-dcc9-11e6-85ec-6818baacb46d.png)

it's right?

**Comment by 4d5452** [comment]
role.json does not declare a "date" type within the "created/modified" properties.

That is:
`
"created":{ "defaultFn": "now"}
`
_When it should be_
`
"created":{ "type": "date", "defaultFn": "now" }
`

**To Fix**:  Extend (or replace) the built in Role model by creating your own role.json file within the common/models directory.  Hope this helps.

**Comment by jdhrivas** [comment]
@FelipeAguayo @4d5452 I created a pull request to update the model.

**Comment by bajtos** [comment]
Fixed by #3161
